cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/query/AbstractIndexTestBase.java

```diff
@@ -43,16 +43,16 @@ public void setUp() throws Exception {
         assertThat(ds, is(notNullValue()));
         im = new IndexManager(ds);
         assertThat(im, is(notNullValue()));
-        assertThat(im.getDatabaseQueue(), is(notNullValue()));
+        assertThat(TestUtils.getDBQueue(im), is(notNullValue()));
```

Comment:
I agree with @ricellis, no need to have `TestUtils.getDBQueue(im)` run multiple times each test.